src/votekit/cleaning.py

```diff
@@ -20,3 +25,90 @@ def remove_empty_ballots(
         pp_clean = PreferenceProfile(ballots=ballots_nonempty)
 
     return pp_clean
+
+
+def clean(
```

Comment:
If this is meant to only be used internally, consider renaming to _clean so a person browsing the code would take the hint that it isn't meant to be used by library users.